scripts/sanitycheck

```diff
@@ -3317,6 +3322,16 @@ def main():
         error("west-flash requires device-testing to be enabled")
         sys.exit(1)
 
+    if os.name == 'nt' and \
+       (not options.build_only or not options.disable_size_report):
+        print("Testing with sanitycheck on Windows is not fully supported yet.")
+        print("Only build without size report works for now:")
+        print("\t- '--build-only --disable-size-report'")
+        print("\t- Requires make: 'choco install make'")
```

Comment:
I don't see why an installation tip for make is buried in an error message about unrelated options.

You already updated the documentation which is great. Now what is the error message for people who already installed their environment earlier and miss make? If it's something like "make not found" then there's nothing to do, you can just remove that line. If the error is more cryptic for some unknown reason then implement a better error message just before the first make invocation:

```python
if make_not_found:
    error("make not found, check the getting started guide")
```